includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -839,7 +839,8 @@ public static function enqueue_styles_scripts() {
 		// Get admin screen id
 		$screen = get_current_screen();
 
-		$is_woocommerce_screen = in_array( $screen->id, array( 'product', 'edit-shop_order', 'shop_order', 'edit-shop_subscription', 'shop_subscription', 'users', 'woocommerce_page_wc-settings', 'woocommerce_page_wc-orders' ), true );
+		$subscriptions_screen_id = wc_get_page_screen_id( 'shop_subscription' );
```

Comment:
This function doesn't exist in older versions of WC. So we'll need to change this to something similar to what we've done in other areas of this PR:

```suggestion
		$subscriptions_screen_id = wcs_is_custom_order_tables_usage_enabled() ? wc_get_page_screen_id( 'shop_subscription' ) : 'shop_subscription';
```

Reply:
Resolved in [c223624](https://github.com/Automattic/woocommerce-subscriptions-core/pull/297/commits/c22362481c344dc7c4e2c389d9f20060005747f9).